deepomatic/api/resources/task.py

```diff
@@ -62,17 +64,14 @@ def list(self, task_ids):
         assert(isinstance(task_ids, list))
         return super(Task, self).list(task_ids=task_ids)
 
-    def wait(self, timeout=60, wait_exp_multiplier=0.05, wait_exp_max=1.0):
+    def wait(self, **retry_kwargs):
         """
         Wait until task is completed. Expires after 'timeout' seconds.
         """
         try:
-            retryer = Retrying(wait=wait_random_exponential(multiplier=wait_exp_multiplier, max=wait_exp_max),
-                               stop=stop_after_delay(timeout),
-                               retry=retry_if_result(is_pending_status),
-                               before=before_log(logger, logging.DEBUG),
-                               after=after_log(logger, logging.DEBUG))
-            retryer(self._refresh_status)
+            retry(self._refresh_status,
+                  retry_if_result(is_pending_status),
```

Comment:
we should probably retry getting the task state if it failed for any reason: we are guarded from infinite retry with this outer retry timeout (default 60s)